libraries/lib-mixer/WideSampleSequence.h

```diff
@@ -47,6 +48,25 @@ class MIXER_API WideSampleSequence
     */
    virtual size_t GetBestBlockSize(sampleCount t) const = 0;
 
+   /*!
+   @param t new cursor position
+   @param playbackDirection whether GetNext should thereafter proceed forwards
+   or backwards
+   */
+   virtual void ResetCursor(double t, PlaybackDirection playbackDirection) = 0;
+
+   /*!
+   Attempts to fill `buffers` with the next available `numSamples`. The progress
+   direction is forward, except if specified otherwise in a prior call to
+   `ResetCursor`. The internal cursor will progress in that direction by
+   `numSamples`.
+   @param buffers output buffer
+   @param numChannels the number of channels in `buffers`. // todo(mhodgkinson) review handling of possible inconsistencies between the requested number of channels and that available in internal clips.
+   @param numSamples the number of samples queried
+   */
+   virtual bool
+   GetNext(float* buffers[], size_t numChannels, size_t numSamples) = 0;
```

Comment:
I see this header file changes in the second commit in the sequence, and nothing else changes in lib-mixer thereafter.

You know I dislike the notion of an internal, stateful cursor leaking out of the abstract interface, and I would think it more elegant instead to change the `Get` function to take a direction argument, and also to specify the reversal of samples is done in the result.

But I think I will not insist on this if the schedule presses.